bc_obps/registration/models/registration_purpose.py

```diff
@@ -0,0 +1,44 @@
+import typing
+from registration.models.time_stamped_model import TimeStampedModel
+from registration.models.operation import Operation
+from django.db import models
+from simple_history.models import HistoricalRecords
+from django.core.cache import cache
+
+
+class RegistrationPurpose(TimeStampedModel):
```

Comment:
Possibly this should be a table of the purposes instead and we have a many-to-many with operations instead of a foreign key